index.bs

```diff
@@ -4629,75 +4618,88 @@ Note: If an error name is not listed here, please file a bug as indicated at the
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="unknownerror" exception export><code>UnknownError</code></dfn>"</td>
             <td>The operation failed for an unknown transient reason (e.g. out of memory).</td>
-
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="constrainterror" exception export><code>ConstraintError</code></dfn>"</td>
             <td>A mutation operation in a transaction failed because a constraint was not satisfied.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="dataerror" exception export><code>DataError</code></dfn>"</td>
             <td>Provided data is inadequate.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="transactioninactiveerror" exception export><code>TransactionInactiveError</code></dfn>"</td>
             <td>A request was placed against a transaction which is currently not active, or which is finished.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="readonlyerror" exception export><code>ReadOnlyError</code></dfn>"</td>
             <td>The mutating operation was attempted in a "readonly" transaction.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="versionerror" exception export><code>VersionError</code></dfn>"</td>
             <td>An attempt was made to open a database using a lower version than the existing version.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="operationerror" exception export><code>OperationError</code></dfn>"</td>
             <td>The operation failed for an operation-specific reason.</td>
             <td>—</td>
         </tr>
         <tr>
-
             <td>"<dfn id="notallowederror" exception export><code>NotAllowedError</code></dfn>"</td>
             <td>The request is not allowed by the user agent or the platform in the current context, possibly because the user denied permission.</td>
             <td>—</td>
         </tr>
     </tbody>
 </table>
 
+<div class=advisement>
+    Additionally, the following error names are kept for legacy reasons but their usage is discouraged:
+
+    <table id="legacy-error-names" class="vert data">
+        <thead>
+            <tr>
+                <th>Name</th>
+                <th>Alternative</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>
+                    "<dfn id="invalidaccesserror" exception export><code>InvalidAccessError</code></dfn>"<br>
+                    <dfn id="dom-domexception-invalid_access_err" for="DOMException" const export><code>INVALID_ACCESS_ERR</code></dfn>&nbsp;(15)
+                </td>
+                <td>
+                    Use <emu-val>TypeError</emu-val> for invalid arguments and
+                    {{NotSupportedError}} for unsupported operations instead.
```

Comment:
Probably want quotes around NotSupportedError. Maybe even `the "NotSupportedError" DOMException`